src/bioregistry/data/bioregistry.json

```diff
@@ -68117,6 +68117,7 @@
       "fairsharing": "FAIRsharing.847069"
     },
     "name": "Open Biological and Biomedical Ontologies",
+    "pattern": "^[A-Za-z][A-Za-z_0-9]+$",
```

Comment:
so can you give some examples of what is actually valid in this URI space since ther aare actually a lot of things tat get put here

Reply:
Yeah I contemplated that again. I thought this should be a fall-through identifier pattern for everything apart from the gravest violations like leading numbers, punctuation etc (for the case when using bioregistry to validate a curie). But now I am thinking that this may not be enough for example if we want to put ontology purls into CURIE syntax: `obo:mondo/mondo-base.owl`.

Maybe there is no point in documenting a pattern? I always thought it was for validation purposes, and therefore useful, but maybe that's not what it is for?

Reply:
Yes, the pattern is for validation, but if we put a pattern that's too strict then we will get upset. Bioregistry has a slot for multiple correct example IDs as well as some example incorrect IDs so we can sanity check the pattern. If you make a list of a couple we can test this pattern, otherwise if it's too much work then this one might not be worth bothering

Reply:
```suggestion
```

I have no meaningful picture of the entire space of possible values - so lets drop it for now and re-introduce it if it becomes needed. Thanks for holding hands!